<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterConfigBase</class>
 <widget class="QWidget" name="FilterConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QCheckBox" name="chkFromList">
     <property name="text">
      <string>&amp;Receive message only from contacts from list</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAuthFromList">
     <property name="text">
      <string>&amp;Receive authorization and system messages only from contacts from list</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblFilter">
     <property name="text">
      <string>Reject messages from unknown users contains this words or phrases:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="edtFilter"/>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel1">
     <property name="text">
      <string>Words are divided by any separators (space, comma, i.e.)
A phrase can be specified using quotation marks
Words can contain wildcards:
* - any amount of symbols (or is empty)
? - any symbol</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
